x-pack/plugins/observability/server/lib/rules/custom_threshold/custom_threshold_executor.ts

```diff
@@ -86,7 +86,6 @@ export const createCustomThresholdExecutor = ({
       executionId,
     });
 
-    // TODO: check if we need to use "savedObjectsClient"=> https://github.com/elastic/kibana/issues/159340
```

Comment:
@fkanout I didn't see any usage of `savedObjectsClient` in this file, so I assume it was cleaned up. Please let me know if I am missing something.

Reply:
I think it is safe to remove it then.